services/cert-manager/0.2.7/root-ca.yaml

```diff
@@ -9,9 +9,6 @@ spec:
   prune: true
   interval: 1m0s
   path: ./services/cert-manager/0.2.7/root-ca
-  dependsOn:
-    - name: cert-manager-release
-      namespace: ${releaseNamespace}
```

Comment:
How does removing this dependency help here? The cert-manager-release will be deployed, anyway?!

Reply:
Currently on the attached clusters that comes with cert-manager(e.g., konvoy2 clusters), the cert-manager-release will fail and block the creation of the root-ca resources.

And yes, the cert-manager-release will be deployed and show as failed anyways and it has been like that. Figuring out how to determine whether cert-manager exists on a cluster is something we want to do but not necessarily just to get through the current state.

Reply:
However, if cluster misses the cert-manager and we drop this depends, customization will try to apply nonexistent CRD. That's why this dependency exists

Reply:
if we are going down this path, lets increase the `timeout` and `interval` to a higher value just in case.

Reply:
@takirala How about bump to the `timeout` to 300s and keep the `interval` as-is?

Reply:
ended up changing to 480s, since cert-manager retries 30 times with 15s interval.